src/vaadin-message-list.js

```diff
@@ -13,10 +13,11 @@ import './vaadin-message.js';
  *
  * ```html
  * <vaadin-message-list
- *   items='[{"text":"Hello list","time":"yesterday", "user": {"name":"Matt Mambo","abbr":"MM","colorIndex":"1"}},{"text":"Hello list","time":"right now", "user": {"name":"Linsey Listy","abbr":"LL","colorIndex":"2"}}]'>
+ *   items='[{"text":"Hello list","time":"yesterday","userName":"Matt Mambo","userAbbr":"MM","userColorIndex":1},
+ *           {"text":"Another message","time":"right now", "userName":"Linsey Listy","userAbbr":"LL","userColorIndex":2, "userImg":"/test/visual/avatars/avatar.jpg"}]'>
  * </vaadin-message-list>
  * ```
- *
+ * It is recommended to set it programmatically by calling messageList.items = [...];
```

Comment:
Same as above, let's provide a more detailed example like the one in `vaadin-menu-bar` API docs.

Reply:
Done.